### DISCUSSION TITLE
Mermaid diagrams aren't rendered properly

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#13362] [Mobile]
Instead of having the diagram that shows up, only its content will be displayed, rendered as a `code` block

**Comment by aconsuegra** [comment]
Hi @EDM115, thanks for the feature request! While we would like to include full support for mermaid diagrams in the app, I think what you propose is a good temporary workaround. I'll create an internal issue to track it and hopefully we can come back to you with some good news. Keep the feedback coming!
